.github/workflows/README.md

```diff
@@ -0,0 +1,142 @@
+      <td><sub>dev-environment</sub></td>
+      <td><sub>Only if you want to deploy on <b>GKE</b> from dev branch to your dev cluster</sub></td>
+    </tr>
+    <tr>
+      <td><sub>GKE_DEV_CLUSTER_ZONE</sub></td>
+      <td><sub>The zone your dev cluster is located</sub></td>
+      <td><sub>us-east1-c</sub></td>
+      <td><sub>Only if you want to deploy on <b>GKE</b> from dev branch to your dev cluster</sub></td>
+    </tr>
+    <tr>
+      <td><sub>GKE_DEV_CLUSTER_PROJECT</sub></td>
+      <td><sub>The name of the GKE project containing your dev cluster</sub></td>
+      <td><sub>your-project</sub></td>
+      <td><sub>Only if you want to deploy on <b>GKE</b> from dev branch to your dev cluster</sub></td>
+    </tr>
+    <tr>
+      <td><sub>GKE_PROD_CLUSTER_NAME</sub></td>
+      <td><sub>The name of your prod cluster</sub></td>
+      <td><sub>prod-environment</sub></td>
+      <td><sub>Only if you want to deploy on <b>GKE</b> from main/master branch to your prod cluster</sub></td>
+    </tr>
+    <tr>
+      <td><sub>GKE_PROD_CLUSTER_ZONE</sub></td>
+      <td><sub>The zone your prod cluster is located</sub></td>
+      <td><sub>us-east1-c</sub></td>
+      <td><sub>Only if you want to deploy on <b>GKE</b> from main/master branch to your prod cluster</sub></td>
+    </tr>
+    <tr>
+      <td><sub>GKE_PROD_CLUSTER_PROJECT</sub></td>
+      <td><sub>The name of the GKE project containing your prod cluster</sub></td>
+      <td><sub>your-project</sub></td>
+      <td><sub>Only if you want to deploy on <b>GKE</b> from main/master branch to your prod cluster</sub></td>
+    </tr>
+    <tr>
+      <td><sub>LKE_SERVICEACCOUNT_TOKEN</sub></td>
+      <td><sub>This is a token an `API Token` (also called `Personal Access Token`) created in a bot account created for service purposes</sub></td>
+      <td><sub>AR4nd0m5tr1ng</sub></td>
+      <td><sub>Only if you want to deploy on <b>LKE</b></sub></td>
+    </tr>
+    <tr>
+      <td><sub>LKE_DEV_CLUSTERID</sub></td>
+      <td><sub>The number in your DEV Linode Kubernetes Cluster URL clusters/*****/summary</sub></td>
+      <td><sub>99999</sub></td>
+      <td><sub>Only if you want to deploy on <b>LKE</b> from dev branch to your dev cluster</sub></td>
+    </tr>
+    <tr>
+      <td><sub>LKE_PROD_CLUSTERID</sub></td>
+      <td><sub>The number in your PROD Linode Kubernetes Cluster URL clusters/*****/summary</sub></td>
+      <td><sub>99999</sub></td>
+      <td><sub>Only if you want to deploy on <b>LKE</b> from main/master branch to your prod cluster</sub></td>
+    </tr>
+  </tbody>
+</table>
+
+### Add Deployment code snippet
+You can add any number of the following snippets to the end of your push-{branch}.yml file to enable the deployment of the selected platform (Secrets must have been populated as describe above). An important step while injecting these code blocks is to make sure that the added job is tabbed like the other jobs already in the workflow file.
+#### GKE's CD script
```

Comment:
Maybe GKE's deployement script?
I don't have a preference between the two, but it it worthwhile considering (same for LKE)

Reply:
I like it, made the changes :-)